requirements/requirements.txt

```diff
@@ -1,7 +1,6 @@
 requests~=2.20.0
-gTTS~=2.2.2
 PyAudio~=0.2.11
-pyee~=8.1.0
+pyee>=8.1.0
```

Comment:
Same as above, should this be ~=8.1?

Reply:
we are not tied to this version, and can use any of the latest ones, however adapt pins this version to 8.1.0 so i can not do ~=8.2 as i originally intended

Reply:
My main concern here is if they release `pyee==9.0.0`, we could potentially update and break things

Reply:
fair enough, the MYCROFT_LOOSE_REQUIREMENTS accounts for this either way, so i will do this change